components/camel-azure-storage-blob/src/main/java/org/apache/camel/component/azure/storage/blob/BlobConsumer.java

```diff
@@ -94,6 +94,7 @@ private Exchange createExchangeFromBlob(final String blobName, final BlobContain
 
         getEndpoint().setResponseOnExchange(response, exchange);
 
+        exchange.getIn().setHeader(BlobConstants.BLOB_NAME, blobName);
```

Comment:
This doesn't seem to be related to it test

Reply:
I guess this info can simplify the tests as I recall the name wasn't populated in the headers but now with regex and batch blob, I guess is needed to have the name in the headers, hence I don't see any problems here